mpf/platforms/pkone/pkone_coil.py

```diff
@@ -149,3 +149,7 @@ def pulse(self, pulse_settings: PulseSettings) -> None:
         cmd = "PCP{}{:02}".format(self.number.board_address_id, self.number.coil_number)
         self.log.debug("Sending Pulse coil command: %s", cmd)
         self.send(cmd)
+
+    def timed_enable(self, pulse_settings: PulseSettings, hold_settings: HoldSettings):
```

Comment:
This could be inside the platform interface. I don't think we have to repeat it everywhere.

Reply:
Thanks, I'll make that change. Wasn't sure what the protocol was, and how much logic an "interface" should have. But it definitely makes it nicer :)

Reply:
Ah, that's what I was afraid of. Prospector doesn't like the change:
```
pylint: abstract-method / Method 'timed_enable' is abstract in class 'DriverPlatformInterface' but is not overridden
```

Would it make sense to have the platform interface method be regular and not abstract? That will solve the problem, but speaks to my above question about what is "allowed" in an interface file. Your thoughts appreciated, thanks!

Reply:
You can create a new partial class or mixing for platforms which need to emulate this "TimedCoilSoftwareEmulation" or something like that.